webapp/src/snippetBuilder.tsx

```diff
@@ -0,0 +1,339 @@
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    . . . . . . . . . . . . . . . . 
+    \`, SpriteKind.Player)`,
+    questions: [
+        {
+            "title": "What should your sprite be called?",
+            "inputs": [{
+                    "answerToken": "spriteName",
+                    "defaultAnswer": "mySprite",
+                    "type": "text"
+            }],
+            "output": "",
+            "goTo": 2
+        },
+        {
+            "title": "Where should your sprite be placed?",
+            "inputs": [
+                {
+                    "label": "x:",
+                    "defaultAnswer": 80,
+                    "answerToken": "xLocation",
+                    "type": "number"
+                },
+                {
+                    "label": "y:",
+                    "defaultAnswer": 60,
+                    "answerToken": "yLocation",
+                    "type": "number"
+                }
+            ],
+            "output": "$spriteName.setPosition($xLocation,$yLocation)",
+            "goTo": 3
+        }, 
+        // {
+        //     "title": "What should your sprite look like?",
+        //     "inputs": [
+        //         {
+        //             "answerToken": "spriteImage",
+        //             "type": "spriteEditor"
+        //         }
+        //     ],
+        //     "ouput": "${spriteName} = sprites.setImage(img```${spriteImage}```)",
+        //     "goTo": 1
+        // },
+        {
+            "title": "What kind of sprite should this be?",
+            "inputs": [
+                {
+                    "answerToken": "spriteKind",
+                    "defaultAnswer": "SpriteKind.Player",
+                    "type": {
+                        "options": [
+                            {
+                                "value": "SpriteKind.Player",
+                                "label": "Player"
+                            },
+                            {
+                                "value": "SpriteKind.Projectile",
+                                "label": "Projectile"
+                            },
+                            {
+                                "value": "SpriteKind.Food",
+                                "label": "Food"
+                            },
+                            {
+                                "value": "SpriteKind.Enemy",
+                                "label": "Enemy"
+                            }
+                        ]
+                    }
+                }
+            ],
+            "output": "$spriteName.setKind($spriteKind)",
+            "goTo": {
+                "4": { "spriteKind": 0 },
```

Comment:
typically in json schemas like this we try to avoid putting values in the "key" slot out of convention.
Maybe you could do:
```
"goTo": {
  "question": 4,
  "parameters": {
     "spriteKind": 0
  }
}
```